From 829a2900c2782bbf4d32299e5f61656466bed012 Mon Sep 17 00:00:00 2001
From: Igal Zeltser <izeltser@habana.ai>
Date: Wed, 13 Dec 2023 14:28:34 +0200
Subject: [PATCH] Revert "[SW-0] gaudi3: increase bar4 size to 128gb"

This reverts commit dc4389ce62642209ab0e8ef6f4cae2ee43f8ce48.
this WA is not required anymore

Change-Id: Ic940d6c19368343318e3715f37b9bfc41472ad80
Signed-off-by: Igal Zeltser <izeltser@habana.ai>
---

diff --git a/arch/gaudi3/arc/common.c b/arch/gaudi3/arc/common.c
index 63d254e..bc58a25 100644
--- a/arch/gaudi3/arc/common.c
+++ b/arch/gaudi3/arc/common.c
@@ -1322,8 +1322,6 @@
 
 		pcie_set_pcie_device_vendor_id();
 
-		preboot_update_bar_sizes();
-
 		if (true) { //(arcboot_rom_global.config_fuse & EFUSE_SYSCFG_PCIE_GEN_LIMIT) == 0) {
 	//		arcboot_rom_status_update(PROGRESS_ARCBOOT_ROM_PCIE_LINK_LIMIT_GEN1);
 			/* limit PCIe link gen to 1 */
